src/ValueHandler/ImmutableSlugValueHandler.php

```diff
@@ -60,7 +60,9 @@ public function handle($subject, string $attribute, array $value): void
         foreach ($value as $item) {
             $localeCode = $item['locale'];
             $valueToSlugify = $item['data'];
-            Assert::string($valueToSlugify);
+            if (!is_string($valueToSlugify)) {
+                continue;
+            }
```

Comment:
I'm not sure about this, maybe throwing an exception is a more secure way to handle this case? It sounds like an error if a value is not a string...